modules/spring-data-commons/src/main/java/org/apache/ignite/springdata/proxy/IgniteClientProxy.java

```diff
@@ -38,4 +39,20 @@ public IgniteClientProxy(IgniteClient cli) {
     @Override public <K, V> IgniteCacheProxy<K, V> cache(String name) {
         return new IgniteCacheClientProxy<>(cli.cache(name));
```

Comment:
We should return `null` in case of cache is `null` (the same way as for `Ignite IgniteProxyImpl`)

Reply:
It seems that the ClientCache instance obtained through the client API cannot be null in the current implementation.